### PR TITLE
Add Wolf Favicons and Enable Social Sharing Previews

### PR DESCRIPTION
**Pull request by stphnwlsh**
* Added to the Gemfile to enable local validation
* Adding a social.html template to include for social site previews
* Adding custom configuration for social previews

### CODE REVIEW

**Comment by stphnwlsh** [comment]
Screen Shot 2022-02-14 at 8 08 02 pm

Tested this locally to validate that it works.  If you publish it you can validate the changes here:  https://www.opengraph.xyz/url/https%3A%2F%2Fmichael-wolfenden.github.io%2F/

**Comment by stphnwlsh** [comment]
@michael-wolfenden Any thoughts on this one?  Happy for you to reject it, just looking to close out my open PR's.  

I thought it would be nice to make the social sharing of the site a little nicer given I share it with my co-workers quite often. 
 The wolf image is from the Android emoji's, happy to make it a different one instead. 
 
 Anyways thanks for all your hard work keeping this up!!!!

**Comment by michael-wolfenden** [comment]
Apologies @stphnwlsh, I dropped the ball and completely missed this pull request. Will have a look soon.

File: Gemfile [hunk 1]
```diff
@@ -7,4 +7,6 @@ group :jekyll_plugins do
   gem 'jekyll-gist'
   gem 'jekyll-paginate'
   gem 'jekyll-algolia'
+  gem 'kramdown-parser-gfm'
+  gem 'webrick'
```
Comment (michael-wolfenden) [inline Gemfile:11]: Are these addition gems required?

Comment (stphnwlsh) [reply Gemfile:11]: I'm have a look but I'm pretty sure it had to do with running Jekyll on my local. Will get back to you on it today

Comment (stphnwlsh) [reply Gemfile:11]: Ah okay I found it!

When I pulled the site to run it locally and make sure the images loaded correctly I got errors when trying to serve the site

```
Dependency Error: Yikes! It looks like you don't have kramdown-parser-gfm or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- kramdown-parser-gfm' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
  Conversion error: Jekyll::Converters::Markdown encountered an error while converting '_posts/2019-01-02-january-2nd-2019.md':
                    kramdown-parser-gfm
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    kramdown-parser-gfm
```

So I added those 2 dependencies and the site generated and previewed just fine

**Comment by stphnwlsh** [comment]
> Apologies @stphnwlsh, I dropped the ball and completely missed this pull request. Will have a look soon.

No need to apologise!  I'm the one messing around with your site.  One thing is I have used the Wolf emoji from Android.  Which might look different when viewing the site on a mac.  Happy to change that up if it looks funny

**Comment by michael-wolfenden** [comment]
Thanks for the pull request @stphnwlsh, much appreciated 🎉

**Pull request merged**